dask_kubernetes/tests/test_core.py

```diff
@@ -313,6 +314,94 @@ def test_scale_up_down(cluster, client):
     assert set(cluster.scheduler.workers) == {b}
 
 
+def test_scale_up_down_fast(cluster, client):
+    cluster.scale(1)
+
+    start = time()
+    while len(cluster.scheduler.workers) != 1:
+        sleep(0.1)
+        assert time() < start + 10
+
+    # Put some data on this worker
+    future = client.submit(lambda: b'\x00' * int(1e6))
+    wait(future)
+
+    # Rescale the cluster many times without waiting: this should put some
+    # pressure on kubernetes but this should never fail nor delete our worker
+    # with the temporary result.
```

Comment:
If you wanted to verify that we never delete the worker with the temporary result then we should probably verify that the following is the same before and after

    cluster.scheduler.tasks[future.key].who_has

Reply:
Thanks, I will do that.